lib/fuse_loop_mt.c

```diff
@@ -164,6 +164,14 @@ static void *fuse_do_work(void *data)
 		pthread_mutex_lock(&mt->lock);
 		if (!isforget)
 			mt->numavail++;
+#if 0
```

Comment:
If it doesn't make sense, then let's just remove the code rather than #ifdef it out.

Reply:
Fine we me as well. But  we cannot just delete or comment out this code here before updating the condition above, as libfuse would other create an indefinite number of threads.